docs/physics/setup/plasma/properties/basic.rst

```diff
@@ -0,0 +1,58 @@
+************************
+Basic General Properties
+************************
+
+After reading in the inputs, TARDIS begins its calculations of the plasma states by first calculating some basic properties that sets a foundation for more advanced calculations later. These properties generally involve properties of atoms and elements. 
+
+Selected Atoms
+==============
+Dentoted as: selected-atoms
+Calculated from: abundance
+
+In order to obtain any calculations regarding an atom, its atomic number must be known, that is, the number of protons within the nucleus of the atom. When the atomic data file is read in via the :code:`atom-data` parameter in the configuration, TARDIS is made aware of the atomic number of every element within the file, so long as the file is formatted correctly, and must chose which atomic numbers correspond to the elements and isotopes stated within the :code:`abundace` field of the configuration. To do this, TARDIS simply in the index of the specified abundance list, giving it the atomic numbers of each element and isotope present within the simulation, while saving these numbers as a `Pandas <https://pandas.pydata.org/docs/>`_ index named :code:`selected-atoms`.
+
+Atomic Mass
+===========
+Denoted as: atomic-mass
+Calculated from: atomic-data, selected-atoms
```

Comment:
these relationships are already present in the code and should be used to generate this.